src/components/Pool/PairsList/Pair/Pair.components.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext, FC } from 'react'
+import { ThemeContext } from 'styled-components'
+import { Text } from 'rebass'
+import { useIsMobileByMedia } from '../../../../hooks/useIsMobileByMedia'
+
+import { ValueWithLabel } from '../../PairView/ValueWithLabel/ValueWithLabel.component'
+
+export const ResponsiveValueWithLabel: FC<{
```

Comment:
why use FC?

Reply:
It's the functional component type (FC) from React. Here I'm defining the FC type receives N props of N types.